install/helm/openchoreo-observability-plane/Chart.yaml

```diff
@@ -26,3 +26,22 @@ keywords:
 maintainers:
   - name: OpenChoreo Team
     url: https://github.com/openchoreo/openchoreo
+
+dependencies:
+  - name: opensearch-operator
+    version: 2.8.0
+    repository: "https://opensearch-project.github.io/opensearch-k8s-operator/"
```

Comment:
We need to use our fork of the operator as this version does not support volumes and volumeMounts. They are required to use CSI secret store driver